tests/clients/test_basic_client.py

```diff
@@ -88,6 +91,8 @@ def test_evaluate_after_fit_enabled() -> None:
 
 def test_evaluate_after_fit_disabled() -> None:
     fl_client = MockBasicClient()
+    fl_client.validate = MagicMock()  # type: ignore
+    fl_client.validate.return_value = fl_client.mock_loss, fl_client.mock_metrics
```

Comment:
Are these values actually required for this test, since you're not asserting that they are equal to anything?

Reply:
these are unfortunately required for the test to run properly